### ISSUE TITLE
plug.proto import not working

### ISSUE THREAD

**Issue by elliotmr** [elliotmr/plug#5]
The import function for plug.proto is not working in the plugin benchmarks here: https://github.com/uberswe/go-plugin-benchmark

See comment: https://old.reddit.com/r/golang/comments/kq1mvq/benchmarking_the_go_plugin_package_vs_other/gi59dgm/

**Comment by elliotmr** [comment]
@uberswe I am having trouble reproducing the issue, I cloned your repo, deleted plug.proto and was able to regenerate everythign using gen.sh.

I noticed that gen.sh was not executable, did you manually generate call protoc rather than using gen.sh?
Also, what verion of protoc-gen-go do you have?

**Comment by uberswe** [comment]
Hey @elliotmr, thanks for looking into it. I did run the gen.sh file using `sh gen.sh` locally and it seems to have executed. I did it again now and I get the following

```
% sh gen.sh 
plugin.proto:3:1: warning: Import plug.proto but not used.
```

It's possible I missed this warning before. I manually copied to plug.proto file into my repository from yours as I assumed it was needed.

I removed the import and it still seems to work so it may have been my fault for not noticing the warning, here is my commit https://github.com/uberswe/go-plugin-benchmark/commit/05407b3f14aec439121b21ebdc654c01a2d5f87b

I guess my implementation for the benchmark is so simple that I don't really need these files `-I=$(go list -f '{{.Dir}}' github.com/elliotmr/plug/pkg/plugpb)`
